README.md

```diff
@@ -61,13 +86,13 @@ az group create -n $PACKER_IMAGE_RESOURCE_GROUP -l $PACKER_IMAGE_LOCATION
 
 # Build the pipeline VM image
 export PACKER_PIPELINE_DOWNLOAD_URL='https://github.com/cisco-ie/pipeline-gnmi/raw/master/bin/pipeline'
-packer build packer/pipelne.json
+packer build pipelne.json

```

Comment:
I know it was a typo from before - but is this supposed to be pipeline.json ? Not pipelne.json